static/src/js/routes/Search/__tests__/Search.test.js

```diff
@@ -104,11 +103,20 @@ describe('Search component', () => {
     expect(portalFeatureContainer.props().advancedSearch).toBeTruthy()
   })
 
+  test('calls onTogglePortalBrowserModal(true) when "Browse Portals" button is clicked', () => {
```

Comment:
Just curious if this was added for anything to do with the tour? Seems like a good test anyway

Reply:
For some reason, it was showing this was missing test coverage on my PR? So I just went ahead and added a test to cover it.